lms/static/sass/views/_login-register.scss

```diff
@@ -130,10 +130,11 @@ $sm-btn-linkedin: #0077b5;
     .toggle-form {
         @include box-sizing(border-box);
         max-width: 650px;
-        min-width: 400px;
+        min-width: 250px;
         margin: 0 auto;
         padding-left: $baseline; // Don't want to override any top or bottom (CR)
-        padding-right: $baseline // Don't want to override any top or bottom (CR)
+        padding-right: $baseline; // Don't want to override any top or bottom (CR)
+        padding-bottom: ($baseline/2);
```

Comment:
I was just wanting you to update the syntax. When assigning 3 of the 4 of the class's sides a padding value it is easier to read with the way I suggest. It's not a new class I am suggesting, just a different way of writing the Sass